content/ch-applications/vqe-molecules.ipynb

```diff
@@ -206,7 +206,7 @@
    "outputs": [],
    "source": [
     "from qiskit import Aer, transpile, assemble\n",
-    "backend = Aer.get_backend(\"qasm_simulator\")\n",
+    "backend = Aer.get_backend(\"aer_simulator\")\n",
```

Comment:
This notebook (`vqe-molecules.ipynb`) isn't updated to the latest version of Qiskit, I think we should revert the changes in this chapter so the code can still be run using Qiskit 0.23.0. Otherwise, the chapter doesn't work for either.

Reply:
Ok let me change it